lockbox-ios/View/SettingListView.swift

```diff
@@ -73,6 +73,9 @@ extension SettingListView {
                 cell.accessibilityCustomActions = cellConfiguration.accessibilityActions
                 cell.accessibilityIdentifier = cellConfiguration.text
 
+                cell.textLabel?.font = UIFont.preferredFont(forTextStyle: .body)
+                cell.detailTextLabel?.font = UIFont.preferredFont(forTextStyle: .caption2)
```

Comment:
Caption2 feels really small. Its correct for the "Send Usage Data" label, but I think unexpected for the "Auto Lock" and "Preferred Browser" settings.